fabtests/pytest/efa/test_rdm.py

```diff
@@ -56,3 +56,21 @@ def test_rdm_atomic(cmdline_args, iteration_type, completion_type):
                             timeout=1800)
     test.run()
 
+
+@pytest.mark.functional
+@pytest.mark.parametrize("environment_variable", ["FI_EFA_FORK_SAFE", "RDMAV_FORK_SAFE"])
+def test_rdm_fork(cmdline_args, completion_type, environment_variable):
```

Comment:
I do not think it belong's to `test_rdm.py`, fork support show be for any endpoint type.  maybe rename `test_fork_support` and put it in `test_fork_support.py`

Reply:
I added a new file named `test_fork.py` to have this test.